packages/expo-updates/android/src/androidTest/java/expo/modules/updates/loader/FileDownloaderTest.java

```diff
@@ -27,30 +27,6 @@ public void setup() {
     context = InstrumentationRegistry.getInstrumentation().getTargetContext();
   }
 
-  @Test
-  public void testCacheControl_LegacyManifest() {
-    HashMap<String, Object> configMap = new HashMap<>();
-    configMap.put("updateUrl", Uri.parse("https://exp.host/@test/test"));
-    configMap.put("runtimeVersion", "1.0");
-    configMap.put("usesLegacyManifest", true);
-    UpdatesConfiguration config = new UpdatesConfiguration().loadValuesFromMap(configMap);
-
-    Request actual = FileDownloader.setHeadersForManifestUrl(config, null, context);
-    Assert.assertEquals("no-cache", actual.header("Cache-Control"));
-  }
-
-  @Test
-  public void testCacheControl_NewManifest() {
-    HashMap<String, Object> configMap = new HashMap<>();
-    configMap.put("updateUrl", Uri.parse("https://exp.host/manifest/00000000-0000-0000-0000-000000000000"));
-    configMap.put("runtimeVersion", "1.0");
-    configMap.put("usesLegacyManifest", false);
-    UpdatesConfiguration config = new UpdatesConfiguration().loadValuesFromMap(configMap);
-
-    Request actual = FileDownloader.setHeadersForManifestUrl(config, null, context);
-    Assert.assertNull(actual.header("Cache-Control"));
-  }
```

Comment:
let's keep both of these tests and just change the assertion in the first one to match the second (now neither request should have a `Cache-Control` header), since we don't want unintentional regressions on this behavior.

Reply:
oh yea, good point.